src/verify.js

```diff
@@ -16,7 +16,7 @@ window.APP = { store };
 
 function Root() {
   return (
-    <IntlProvider locale={lang} messages={messages}>
+    <IntlProvider locale={getLocale()} messages={getMessages()}>
```

Comment:
I noticed you moved the IntlProvider from Root to the HomePage https://github.com/mozilla/hubs/pull/2879/files#diff-1fdf421c05c1140f6d71444ea2b27638R17 but didn't make the same refactor to the Roots containing SignInPage or VerifyPage . Was the point of the refactor just to move the IntlProvider closer to the code using it? Also -- a question related to React Contexts - is it a bug if we accidentally had an IntlProvider nested within an IntlProvider or is that OK?